code/modules/surgery/organs/organ_icon.dm

```diff
@@ -171,10 +171,10 @@ var/global/list/limb_icon_cache = list()
 			icon_file = 'icons/mob/human_races/robotic.dmi'
 		else
 			if(status & ORGAN_MUTATED)
-				icon_file = species.deform
+				icon_file = owner ? owner.deform : species.deform
 			else
 				// Congratulations, you are normal
-				icon_file = species.icobase
+				icon_file = owner ? owner.icobase : species.icobase
```

Comment:
I thought that issue would crop up if I transplanted a limb from one Vox to another since their icobases are different (different icon skin-tones) but I'll test this on inter-species transplants and get back to you